File: js/ejer_33.js
const rawInput = typeof prompt === 'function'
  ? prompt('Ingrese un número para mostrar su tabla de multiplicar (1-10):')
  : null;
const numero = parseInt(rawInput, 10);

if (!isNaN(numero)) {
  for (let i = 1; i <= 10; i++) {
    console.log(`${numero} x ${i} = ${numero * i}`);
  }
} else {
  console.log('Número inválido.');
}

File: python/ejer_33.py
raw_input_value = input('Ingrese un número para mostrar su tabla de multiplicar (1-10): ')

try:
    numero = int(raw_input_value)
    for i in range(1, 11):
        print(f'{numero} x {i} = {numero * i}')
except ValueError:
    print('Número inválido.')